src/internal.c

```diff
@@ -1883,6 +1883,15 @@ int InitSSL_Ctx(WOLFSSL_CTX* ctx, WOLFSSL_METHOD* method, void* heap)
     return ret;
 }
 
+#ifdef HAVE_EX_DATA_CLEANUP_HOOKS
+void wolfSSL_CRYPTO_cleanup_ex_data(WOLFSSL_CRYPTO_EX_DATA* ex_data, int n_ex_data)
+{
+    for (--n_ex_data; n_ex_data >= 0; --n_ex_data) {
```

Comment:
Can you add a check to ensure n_ex_data does not exceed MAX_EX_DATA?

Reply:
I did one better and refactored `wolfSSL_CRYPTO_cleanup_ex_data()` to take only the `ex_data` arg.  In the current setup it iterates based on the number of array elements in the struct.  If that number becomes dynamic in the future, it can iterate on the length recorded in the struct.